scripts/inline_comments_data_collection.py

```diff
@@ -18,6 +18,12 @@
 
 api = PhabricatorAPI(get_secret("PHABRICATOR_TOKEN"))
 
+db.register(
+    phabricator.REVISIONS_DB,
+    "https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_revisions.latest/artifacts/public/revisions.json.zst",
+    4,
+)
+
```

Comment:
It is already registered:

https://github.com/mozilla/bugbug/blob/c98b7078911f442a99c22407bcd4ffc9beaa59e8/bugbug/phabricator.py#L30-L35

However, there is a bug in the registration; it should register `FIXED_COMMENTS_DB` instead of `REVISIONS_DB.`

Reply:
I see, thank you for pointing that out! I have fixed it here: 5542e679a8634432ff3d5a594a67d50b9317def9.